### PR TITLE
feat(#1816): width property for container

### PR DESCRIPTION
**Pull request by bdfranck**
This PR documents the change from PR https://github.com/GovAlta/ui-components/pull/1851 for issue https://github.com/GovAlta/ui-components/issues/1816. It includes the following changes:

- Adds `Width` property to container playground
- Adds `Width` property description
- Fixes punctuation and spelling for other container property descriptions.

container-width

### CODE REVIEW

**Comment by bdfranck** [comment]
This PR is blocked by https://github.com/GovAlta/ui-components/pull/1851. I'll re-open it once those changes are merged to `alpha` and deployed.

**Comment by bdfranck** [comment]
Instead, I'll re-open it and indicate in the title that it isn't ready for testing.

**Comment by bdfranck** [comment]
Testing note: Viewing the changes requires the latest `alpha` version of the component packages. If you aren't seeing the change in your local environment, try updating your packages with `npm i`.

**Comment by lizhuomeng71** [comment]
issue is verified for react and angular, for chrome, firefox and edge

image

**Comment by bdfranck** [comment]
I've resolved the merged conflicts on my branch.

**Review by chrisolsen — APPROVED** [review]

**Pull request merged**
